docs/Ecosystem.md

```diff
@@ -72,11 +72,11 @@ Plugins maintained by the fastify team are listed under [Core](#core) while plug
 - [`fastify-explorer`](https://github.com/Eomm/fastify-explorer) Get control of your decorators across all the encapsulated contexts
 - [`fastify-esso`](https://github.com/patrickpissurno/fastify-esso) The easiest authentication plugin for Fastify, with built-in support for Single sign-on (and great documentation)
 - [`fastify-favicon`](https://github.com/smartiniOnGitHub/fastify-favicon) Fastify plugin to serve default favicon.
+- [`fastify-gcloud-trace`](https://https://github.com/mkinoshi/fastify-gcloud-trace) [Google Cloud Trace API](https://cloud.google.com/trace/docs/reference) Connector for Fastify.
```

Comment:
The previous commit was correct. The order is:

- `fastify-firestore`
- `fastify-gcloud-trace`
- `fastify-google-cloud-storage`

Reply:
Yeah the last one fixed it